src/View/Requirements_Backend.php

```diff
@@ -643,7 +643,7 @@ public function css($file, $media = null)
     protected function parseModuleResourceReference($file)
     {
         // String of the form vendor/package:resource. Excludes "http://bla" as that's an absolute URL
-        if (preg_match('#([^ ]*/[^ ]*) *: *([^ ]*)#', $file, $matches)) {
+        if (preg_match('#([^\/\/][^ /]*\/[^ /]*) *: *([^ ]*)#', $file, $matches)) {
```

Comment:
Do `[^\/\/]` and `[^\/]` differ at all? It's the same character repeated twice in a char list.